### ISSUE TITLE
Odd MQTT messages being published by the firmware

### ISSUE THREAD

**Issue by diegodorado** [DFRobot/Obloq#2]
The is a strange behaviour i would like to avoid. The firmware seems to publish periodically to the topic H1v67bw3 with a message of 1. When I try it locally, although annoying, I can just ignore that topic. But the server I want to use it against (adafruit.com) disconnects me because I dont have permission to publish on topic H1v67bw3 .
Is there any workaround to overcome this? Some special message I can send to the module to avoid this?
Thanks

**Comment by adumont** [comment]
Hi @diegodorado , it looks a lot like what is doing this [example sketch](https://github.com/DFRobot/Obloq/blob/master/examples/SmartButton/SmartButton.ino). In the example, is trying to send a 1 or 0 (depening on the state of a button) to devTopic = "HkfVZ9BBz". Although not the same topic, it's very similar.

Are you sure it's the Obloq itself and not the Arduino (or other microcontroller) connected to it that is sending that?

**Comment by diegodorado** [comment]
No, it was not the case. It seemed a ping/keepalive/heartbeat message hardcoded on the firmware.
Anyway I do not have the hardware anymore
